### ISSUE TITLE
[BUG] Uploading .jpg - "Not compatible with the selected LLM or not supported. Please select another format."

### ISSUE THREAD

**Issue by Pervect66** [enricoros/big-agi#540]
### Description

Drag&drop any .jpg results in "Not compatible with the selected LLM or not supported. Please
select another format." error.
LMM GPT-40 (2024-05-13) & GPT-4 Turbo (2024-04-09)

### Device and browser

Windows 11 Pro version 23H2 (OS Build 22631.3527)
Intel NUC10 i7 / 64GB RAM
Firefox 125.0.3 (64-bit)
big-AGI 1.16.1

### Screenshots and more

![image](https://github.com/enricoros/big-AGI/assets/118598486/84d81587-f471-480a-8110-a1b54557db9d)

### Willingness to Contribute

- [ ] 🙋‍♂️ Yes, I would like to contribute a fix.

**Comment by enricoros** [comment]
Confirmed

**Comment by huseman21** [comment]
I get the same error.  Was a fix ever developed?

**Comment by enricoros** [comment]
@huseman21 yes It is. If models report their capabilities correctly, then you'll be able to attach images. Moreover, you can still send it. It's just gonna ask you to confirm, period. Consider this as a warning only.

Which Vendor / models are you using, so I can look deeper into this?
